### PR TITLE
Add hello.py

### PR DESCRIPTION
**Pull request by rocco8773**
this defines the `say_hello()` function

### CODE REVIEW

File: hello.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import sys
+
+
+def say_hello(name):
+    print(f"H, {name}!")
```
Comment (smithsp) [inline hello.py:5]: I agree that you define the function here.  Can you add a docstring?

File: hello.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+
+if __name__ == "__main__":
+    if len(sys.argv) > 1:
+        name = sys.arv[1]
```
Comment (smithsp) [inline hello.py:10]: Did you test this?
```suggestion
        name = sys.argv[1]
```

**Review by smithsp — CHANGES_REQUESTED** [review]
@rocco8773 This is great to add.  Thanks for your contribution. Please see the detailed comments.

File: hello.py (hunk 1 above)
Comment (rocco8773) [reply hello.py:5]: Totally agree, I'll add this!

File: hello.py (hunk 2 above)
Comment (rocco8773) [reply hello.py:10]: Yep, you're correct.

**Review by StanczakDominik — APPROVED** [review]
And sometimes someone else's procrastinating other work and has time to do a review! Looks great!

File: hello.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import sys
+
+
+def say_hello(name):
+    """Print a hello statement to screen."""
+    print(f"H, {name}!")
```
Comment (smithsp) [inline hello.py:6]:
```suggestion
    print(f"Hi, {name}!")
```

**Review by smithsp — APPROVED** [review]
Looks great!

**Pull request merged**